docs/section22/Rule22-44.md

```diff
@@ -0,0 +1,59 @@
+# CHW & CW - Rule 22-44  
+
+**Rule ID:** 22-44   
+**Rule Description:** Where part-load performance of chillers in the proposed design is not available, and the design temperature across the condenser is 10°F, the performance curves in Normative Appendix J, as referenced in Table J-1, shall be modeled for the specified chiller.  
+**Rule Assertion:** Proposed RMD = expected value  
+**Appendix G Section:** Table G3.1 #10b Proposed Column 
+**Applicability Check**  
+1. Proposed includes at least one non process chiller 
+
+**Mandatory Rule:** True  
+**Evaluation Context:** Each Chiller  
+**Table Lookup:** 90.1 Appendix J Table J-1 
+**Function Call:**  
+1. does_chiller_performance_match_curve()
+
+## Applicability Check:  
```

Comment:
What if it is an air cooled chiller. Does this rule fall under NOT_APPLICABLE perhaps?